scripts/package-cli

```diff
@@ -18,6 +18,7 @@ for i in bandwidth bridge firewall flannel host-local loopback portmap; do
 done
 
 cp contrib/util/check-config.sh bin/check-config
+cp contrib/util/k3s-killall.sh bin/test
```

Comment:
Oh, this is how you've created a forkbomb. 

 /bin/test is an existing command, that is used to implement things like `if [ -n "$var"]` and other "tests". You've overwritten that with this script.

```suggestion
cp contrib/util/k3s-killall.sh bin/killall
```

Reply:
`cp contrib/util/k3s-killall.sh bin/killall`
i believe name it killall will also override a the native killall comand, any other suggestion? k3s-killall?

Reply:
Hmm fair point. I think `k3s-killall` is fine, I just think we should keep the subcommand as `k3s killall`.